### PR TITLE
Support for laravel-mix 2.1.*

### PR DESCRIPTION
**Pull request by cjmaxik**
- Support for laravel-mix 2.1.* (issue #22)
- Component-based approach

Will add some tests in the future

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,37 +1,40 @@
...
-    "laravel-mix": "^1.7.0",
-    "prettier": "^1.10.1"
-  },
-  "peerDependencies": {
-    "laravel-mix": "^2.0.0"
-  },
-  "engines": {
-    "node": ">=6.0.0"
-  }
+    "name": "laravel-mix-purgecss",
```
Comment (sebastiandedeyne) [inline package.json]: Could you revert this indentation change?

Comment (cjmaxik) [reply package.json]: Done

**Comment by cjmaxik** [comment]
@Rias500 it doesn't work somehow... I did the same thing as in https://github.com/JeffreyWay/laravel-mix/blob/master/src/components/PurifyCss.js here

**Comment by Rias500** [comment]
Ah makes sense, hadn't tested it, hope it gets merged soon 👍

**Comment by sebastiandedeyne** [comment]
Gonna merge and release tomorrow :)

**Comment by tpraxl** [comment]
@sebastiandedeyne Thanks for merging. What's your plan for creating a release?

**Comment by sebastiandedeyne** [comment]
Released! 🎉

**Pull request merged**